Documentation/CodeSnippets/Examples/Http/MeowInformationRequester.rst.txt

```diff
@@ -8,6 +8,10 @@
    
    declare(strict_types=1);
    
+   /*
```

Comment:
It is possible to avoid this=

Reply:
What exactly do you want to avoid?

Reply:
The comment part.

Reply:
You want to display it completely or not at all?

Reply:
Not at all. Why show a comment which is useless for this example?

Reply:
My idea was to raise awareness that it should be there. We show a complete file here as opposed to just an excerpt of a PHP class. Same thing with showing `declare(strict_types=1);`  It doesn't play a role in the example but it is best practice. 

If you want to display only relevant parts and not a complete file I would suggest to use `createPhpClassCodeSnippet` instead of `createCodeSnippet` But it would also be possible to introduce another flag that removes instead of replaces the comment.

Reply:
Why should ir ber in the class as comment? The license file is in the repo, that should be enough. But I am not in that legal stuff. Do you have any reference?
But for me it is okay for now.

Reply:
I have no more then the link from TYPO3 explained that lead us to include this...